tests/test_on_data/behavior/test_lightningpose_converter.py

```diff
@@ -80,18 +80,24 @@ def setUpClass(cls) -> None:
         cls.converter_metadata = dict(
             Behavior=dict(
                 PoseEstimation=cls.pose_estimation_metadata,
-                Videos=[
```

Comment:
Similarly for the tests, I'd like to check both the old video metadata structure and the new one to make sure that they both work. Maybe add a TODO comment to remove the old video tests metadata structure tests after we remove the dependence on the old video interface.